### ISSUE TITLE
[PHP] Unable to add file to DescriptorPool

### ISSUE THREAD

**Issue by tangquoctuan** [protocolbuffers/protobuf#5505]
**What version of protobuf and what language are you using?**
Version: dev-master.
Language: PHP

**What operating system (Linux, Windows, ...) and version?**
Linux, Alpine

It's run normally on my local, but when rebuild it on server. I got error when create new request. The error like:

Error alert
`Unable to add file to DescriptorPool: couldn't resolve name '.google.rpc.Status in message 'settings.Config' in /var/www/admin/vendor/Grpc/GPBMetadata/SettingsService/SettingsService.php on line 61`

Generated file:
`Config`
```
<?php
# Generated by the protocol buffer compiler.  DO NOT EDIT!
# source: settings_service/settings_service.proto

namespace Settings;

use Google\Protobuf\Internal\GPBType;
use Google\Protobuf\Internal\RepeatedField;
use Google\Protobuf\Internal\GPBUtil;
use Google\Protobuf\Internal\GPBWrapperUtils;

/**
 * Generated from protobuf message <code>settings.Config</code>
 */
class Config extends \Google\Protobuf\Internal\Message
{
    /**
     * Generated from protobuf field <code>string group = 1;</code>
     */
    private $group = '';
    /**
     * Generated from protobuf field <code>string name = 2;</code>
     */
    private $name = '';
    /**
     *    google.rpc.Status status = 4;
     *
     * Generated from protobuf field <code>string value = 3;</code>
     */
    private $value = '';

    /**
     * Constructor.
     *
     * @param array $data {
     *     Optional. Data for populating the Message object.
     *
     *     @type string $group
     *     @type string $name
     *     @type string $value
     *              google.rpc.Status status = 4;
     * }
     */
    public function __construct($data = NULL) {
        \GPBMetadata\SettingsService\SettingsService::initOnce();
        parent::__construct($data);
    }

    /**
     * Generated from protobuf field <code>string group = 1;</code>
     * @return string
     */
    public function getGroup()
    {
        return $this->group;
    }

    /**
     * Generated from protobuf field <code>string group = 1;</code>
     * @param string $var
     * @return $this
     */
    public function setGroup($var)
    {
        GPBUtil::checkString($var, True);
        $this->group = $var;

        return $this;
    }

    /**
     * Generated from protobuf field <code>string name = 2;</code>
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * Generated from protobuf field <code>string name = 2;</code>
     * @param string $var
     * @return $this
     */
    public function setName($var)
    {
        GPBUtil::checkString($var, True);
        $this->name = $var;

        return $this;
    }

    /**
     *    google.rpc.Status status = 4;
     *
     * Generated from protobuf field <code>string value = 3;</code>
     * @return string
     */
    public function getValue()
    {
        return $this->value;
    }

    /**
     *    google.rpc.Status status = 4;
     *
     * Generated from protobuf field <code>string value = 3;</code>
     * @param string $var
     * @return $this
     */
    public function setValue($var)
    {
        GPBUtil::checkString($var, True);
        $this->value = $var;

        return $this;
    }

}

```
`SettingsService`
```
public static function initOnce() {
        $pool = \Google\Protobuf\Internal\DescriptorPool::getGeneratedPool();
        if (static::$is_initialized == true) {
          return;
        }
        \GPBMetadata\Google\Protobuf\GPBEmpty::initOnce();
        $pool->internalAddGeneratedFile(hex2bin(
            "0ab3090a2773657474696e67735f736572766963652f73657474696e6773" .
            "5f736572766963652e70726f746f120873657474696e67731a17676f6f67" .
            "6c652f7270632f7374617475732e70726f746f224a0a114c697374436f6e" .
            "66696752657175657374120e0a06706172656e7418012001280912110a09" .
            "706167655f73697a6518022001280912120a0a706167655f746f6b656e18" .
            "03200128092298010a124c697374436f6e666967526573706f6e73651238" .
            "0a06636f6e66696718012003280b32282e73657474696e67732e4c697374" .
            "436f6e666967526573706f6e73652e436f6e666967456e7472791a480a0b" .
            "436f6e666967456e747279120b0a036b657918012001280912280a057661" .
            "6c756518022001280b32192e73657474696e67732e436f6e66696742794e" .
            "616d654d61703a02380122a6010a194c697374436f6e666967427947726f" .
            "7570526573706f6e7365123f0a06636f6e66696718012003280b322f2e73" .
            "657474696e67732e4c697374436f6e666967427947726f7570526573706f" .
            "6e73652e436f6e666967456e7472791a480a0b436f6e666967456e747279" .
            "120b0a036b657918012001280912280a0576616c756518022001280b3219" .
            "2e73657474696e67732e436f6e66696742794e616d654d61703a02380122" .
            "ad010a0f436f6e66696742794e616d654d617012350a06636f6e66696718" .
            "012003280b32252e73657474696e67732e436f6e66696742794e616d654d" .
            "61702e436f6e666967456e74727912220a0673746174757318022001280b" .
            "32122e676f6f676c652e7270632e5374617475731a3f0a0b436f6e666967" .
            "456e747279120b0a036b6579180120012809121f0a0576616c7565180220" .
            "01280b32102e73657474696e67732e436f6e6669673a02380122580a0643" .
            "6f6e666967120d0a0567726f7570180120012809120c0a046e616d651802" .
            "20012809120d0a0576616c756518032001280912220a0673746174757318" .
            "042001280b32122e676f6f676c652e7270632e537461747573222f0a1047" .
            "6574436f6e66696752657175657374120d0a0567726f7570180120012809" .
            "120c0a046e616d6518022001280922120a10536574436f6e666967526571" .
            "7565737432fd020a0f53657474696e67735365727669636512490a0a4c69" .
            "7374436f6e666967121b2e73657474696e67732e4c697374436f6e666967" .
            "526571756573741a1c2e73657474696e67732e4c697374436f6e66696752" .
            "6573706f6e7365220012340a0c437265617465436f6e66696712102e7365" .
            "7474696e67732e436f6e6669671a102e73657474696e67732e436f6e6669" .
            "67220012370a0f476574436f6e66696742794e616d6512102e7365747469" .
            "6e67732e436f6e6669671a102e73657474696e67732e436f6e6669672200" .
            "12410a10476574436f6e666967427947726f757012102e73657474696e67" .
            "732e436f6e6669671a192e73657474696e67732e436f6e66696742794e61" .
            "6d654d6170220012310a09536574436f6e66696712102e73657474696e67" .
            "732e436f6e6669671a102e73657474696e67732e436f6e6669672200123a" .
            "0a0c44656c657465436f6e66696712102e73657474696e67732e436f6e66" .
            "69671a162e676f6f676c652e70726f746f6275662e456d70747922006206" .
            "70726f746f33"
        )); <--- line 61 is here

        static::$is_initialized = true;
    }
```
I also found this bug has reported in https://github.com/protocolbuffers/protobuf/issues/3966. But it still error.

**Comment by TeBoring** [comment]
I think #3966 has been fixed by #4139.
Are you using c extension and what version are you using?

**Comment by TeBoring** [comment]
It looks like your error is not the same as #3966.
```.google.rpc.Status``` is not maintained by us. Please contact its owner to autoload it properly.
